File: Alternativas/ej1.py
# -*- coding: utf-8 -*-

#Algoritmo que pida dos números e indique si el primero es mayor que el segundo o no.

#1.almacenar numero1 (real)
#2.almacenar numero2 (real)
#3.restar ambos números
#4.si el resultado es mayor de 0, numero1 es mayor
#5.en caso contrario numero2 es el mayor
#6.si el resultado es 0, son iguales
#7.mostrar el mayor

print("---Introduce dos números y te dire el mayor")
numero1 = float(input("Introduce el primer número> "))
numero2 = float(input("Introduce el segundo número> "))
if((numero1-numero2)>0):
	print("El mayor es ",numero1)
elif((numero1-numero2)<0):
	print("El mayor es ",numero2)
else:
	print("Ambos números son iguales")

File: Java/Alternativas/ej1.java
import java.util.Scanner; //Importar la clase Scanner 

/**
 * Algoritmo que pida dos números e indique si el primero es mayor que el segundo o no.
 * @author Alberto Miguel Martinez Jimenez
 *
 */
/**
 * Algoritmo:
 * Almacena numero1
 * Almacena numero2
 * Comparar si numero1 es mayor que numero2
 * Si es verdadero numero1 es mayor
 * de lo contrario es menor
 * 
 */
public class ej1 {

	public static void main(String[] args) {
		double numero1 = 0;
		double numero2 = 0;
		Scanner scanner = new Scanner(System.in);//Intanciar el objeto Scanner	
		
		System.out.print("Introduce primer número> ");
		numero1 = scanner.nextDouble(); 
		
		System.out.print("Introduce el segundo número> ");
		numero2 = scanner.nextDouble();	
		
		if(numero1 > numero2) {
			System.out.println("El primer número es mayor.");
		}else if(numero1 < numero2) {
			System.out.println("El segundo número es el mayor.");
		}
	}

}
